circle.yml

```diff
@@ -19,7 +19,9 @@ deployment:
         EOF
       - chmod 600 ~/.netrc
       - "[[ ! -s \"$(git rev-parse --git-dir)/shallow\" ]] || git fetch --unshallow"
-      - NODE_ENV=production make deploy env=staging
+      - node_modules/.bin/bucket-assets -b positron-staging
+      - heroku config:set COMMIT_HASH=$(shell git rev-parse --short HEAD) --app=positron-staging
```

Comment:
This should use ASSET_MANIFEST now like [this](https://github.com/artsy/force/blob/master/scripts/deploy.sh#L9)

`heroku config:set ASSET_MANIFEST=$(cat manifest.json)`